plugins/faustwp/includes/replacement/functions.php

```diff
@@ -131,3 +131,106 @@ function is_wp_link_ajax_request(): bool {
 		&& ! empty( $_POST['action'] )
 		&& 'wp-link-ajax' === $_POST['action'] );
 }
+
+/**
+ * Get all site URLs for each possible HTTP protocol
+ */
+function faustwp_get_wp_site_urls() {
+
+	$site_url = site_url();
+	$host_url = wp_parse_url( $site_url, PHP_URL_HOST );
+
+	if ( ! is_string( $host_url ) ) {
+		return apply_filters( 'faustwp_get_wp_site_urls', array( $site_url ) );
+	}
+
+	$is_https = substr( $site_url, 0, 6 ) === 'https:';
```

Comment:
`strpos` can be faster for same purpose.

Reply:
Thanks good to know 😄